src/drivers/Cst816s.cpp

```diff
@@ -32,67 +32,47 @@ void Cst816S::Init() {
   vTaskDelay(5);
   twiMaster.Read(twiAddress, 0xa7, &dummy, 1);
 
+  /*
+  [2] EnConLR - Continuous operation can slide around
+  [1] EnConUD - Slide up and down to enable continuous operation
+  [0] EnDClick - Enable Double-click action
+  */
+  static constexpr uint8_t motionMask = 0b00000101;
```

Comment:
Did you find more detailed info about the CST816S? 
Does this register enables functionalities that weren't working before?

Reply:
Yes, this is for double tap to work and for continuous reading.

Reply:
Maybe we can enable multitouch using this register? I'm not sure we need it, but I remember I've never manager to get more than one touch point from the touch panel.

Reply:
I didn't see anything in the documentation about it, what I realized was that the double touch didn't work and I went to look at the documentation and I found this information, which also indicated the continuous reading mode and activated to try to solve the problem when it was always reading the touch.

Reply:
> and activated to try to solve the problem when it was always reading the touch

Does it actually fix these freezes, by any chance?

Reply:
From what I tested, I was being able to play pong for several minutes without blocking, what I realized was that the touch is sometimes unresponsive, but just lift your finger to get it working again, but it doesn't block the application.

Reply:
I think the touch panel automatically goes to sleep mode after a while. That's maybe why it would appear unresponsive from time to time.

Reply:
Mhm I noticed a crash while playing 2048, but I could play paddle for a few minute without any crash!

Reply:
I think it must not be related to touch. Debug time...

Reply:
Right, 2048 does not enable the continuous mode, so that's probably another bug !